### PR TITLE
Adding Somesh Folder and All Assignments

### PR DESCRIPTION
**Pull request by somesh6797**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Yogesh/HTML/Assig.1/Que2.html
```
Comment (sourabh-kanwade) [inline .gitignore]: Why you added this file

File: Somesh/HTML/Assignment1/que4Assign1.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        </p>
+        <p>
+            I love to solve the problems, but sometimes problems are irritating. I love to watch hollywood movies. I had started to learn to draw recently, it's not my forte but i can draw pictures by watching or following the images.
+        </p>
+        <br />
+        <br />
+        <br />
+    </main>
+    <hr>
+    <footer>
```
Comment (sourabh-kanwade) [inline Somesh/HTML/Assignment1/que4Assign1.html]: Is it required ?

File: .gitignore (hunk 1 above)
Comment (somesh6797) [reply .gitignore]: it was hidden file so got added unknowingly

File: Somesh/HTML/Assignment1/que4Assign1.html (hunk 2 above)
Comment (somesh6797) [reply Somesh/HTML/Assignment1/que4Assign1.html]: no it is not required.

**Review by sourabh-kanwade — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (somesh6797) [reply .gitignore]: deleting it

File: Somesh/HTML/Assignment2/que1Assign2.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+<html lang="en">
+    <head>
+        <title>Document</title>
+    </head>
+    <body style="background-color: rgb(230, 0, 69);">
+        
+            <h1 style="text-align: center;">
+                Google
+            </h1>
+
```
Comment (dayanandviveki) [inline Somesh/HTML/Assignment2/que1Assign2.html]: Remove extra space

**Review by dayanandviveki — APPROVED** [review]
No Changes Required in all assignments

File: Somesh/HTML/Assignment2/que1Assign2.html (hunk 3 above)
Comment (somesh6797) [reply Somesh/HTML/Assignment2/que1Assign2.html]: Removed Changes

**Review by kalyani1942 — APPROVED** [review]

**Review by Prashant698 — APPROVED** [review]

**Review by somesh6797 — COMMENTED** [review]
done

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
add new line at the end of files
indentation is missing in some css code make it correct

File: README.md [hunk 4]
```diff
@@ -1,2 +1 @@
 # MI-11-MEAN
```
Comment (Pawan1005) [inline README.md]: remove this file

File: Somesh/HTML/Assignment1/que4Assign1.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        <p>
+            I love to solve the problems, but sometimes problems are irritating. I love to watch hollywood movies. I had started to learn to draw recently, it's not my forte but i can draw pictures by watching or following the images.
+        </p>
+        <br />
+        <br />
+        <br />
+    </main>
+    <hr>
+</body>
+</html>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment1/que4Assign1.html:41]: need indentation
add new line at the end of file

File: Somesh/HTML/Assignment3/que1Assign3.html [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+                        <ul style="list-style-type: disc;">
+                            <li>Triveni</li>
+                            <li>Kalyani</li>
+                        </ul>
+                    </li>
+                </ol>
+            </li>
+        </ol>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment3/que1Assign3.html:67]: need indentation

File: Somesh/HTML/Assignment3/website/contact-us.html [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+                    <li><a href="contact-us.html">Contact-Us </a></li>
+                </ul>
+            </nav>
+            <main class="main">
+                <h2>Contact-Us!!!!</h2>
+                <img src="./goku.png" alt="">
+            </main>
+        </header>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment3/website/contact-us.html:87]: need indentation

File: Somesh/HTML/Assignment3/website/home.html [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+                </p>
+                <img src="./goku.png" class="right-image" alt="">
+            </div>
+            <div class="section2">
+                <h3>Homepage Subheading</h3>
+                <br>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aspernatur distinctio non veritatis delectus autem facere numquam eius ad, culpa odit illo quaerat tempora dolor officia corrupti eligendi ratione, dolorem quibusdam. Officiis laborum natus dicta, neque itaque nam? Perspiciatis, perferendis et?</p>
+            </div>
+        </main>
+
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment3/website/home.html]: remove extra line

File: Somesh/HTML/Assignment3/website/home.html [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+            user-select: none;
+        }
+        .left-image{
+            transform: scaleX(-1);
+        }
+        p{
+            text-align: justify;
+        }
+    </style>
+</head>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment3/website/home.html]: indentation missing

File: Somesh/HTML/Assignment3/website/mi-mean.html [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+                <br>
+                <ol>
+                    <li>Bipin</li>
+                    <li>Priyanka</li>
+                    <li>Yashasvi</li>
+                </ol>
+            </main>
+        </header>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment3/website/mi-mean.html:89]: indentation missing for head tag

File: Somesh/HTML/Assignment4/que1Assign4.html [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+                    <th>Females</th>
+                    <td>1.7</td>
+                    <td>0.002</td>
+                    <td>43%</td>
+                </tr>
+
+            </caption>
+        </table>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment4/que1Assign4.html:51]: indentation missing for head tag

File: Somesh/HTML/Assignment4/que2Assign4.html [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+            <tr>
+                <th>4:00</th>
+            </tr>
+            <tr>
+                <th>4:30</th>
+                <td></td>
+            </tr>
+        </table>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment4/que2Assign4.html:78]: indentation missing for head tag

File: Somesh/HTML/Assignment4/que3Assign4.html [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+        </tr>
+        <tr>
+             <td>Wednedday</td>
+             <td class="yellow">8.00 a.m.</td>
+             <td class="green">12.00 p.m.</td>
+             <td>XSL Formatting Objects</td>
+         </tr>
+        </table>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Somesh/HTML/Assignment4/que3Assign4.html:75]: indentation missing for head tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**